modal/Modal/Modal.html

```diff
@@ -0,0 +1,158 @@
+                                        class="code-snippet-iframe"
+                                        sandbox="allow-scripts allow-same-origin">
+                                  </iframe>
+                                </div>
+                            </div>
+                        </div>
+        <!-- container component closed-->  
+                  </div>
+              </div>
+<!-- container component aside -->
+<aside class="component-sidebar-container margin-top-right-bottom-left">
+    <div class="sidebar-onthispage-content"> 
+    On this page <br/><br/>
+    <ul class="flex-col row-gap-1rem">
+        <li> <a href="#simple-link-sidebar" class="color-black sidebar-onhover-primary"> Simple Modal </a></li>
+        <li> <a href="#actionmodal-link-sidebar" class="color-black sidebar-onhover-primary"> Action Modal </a> </li>
+    </ul>
+    </div>
+</aside>
+<!-- container closed-->  
+</div>
+<script src="Modal.js"> </script>
+</body>
+</html>
```

Comment:
you can format this piece of code

Reply:
![modalimg](https://user-images.githubusercontent.com/28673856/154227887-e49c587a-b76c-4741-b3be-ff9898babd00.png)